packages/bodiless-components/src/Meta/Meta.tsx

```diff
@@ -106,6 +107,17 @@ const withMetaHtml = (
   );
 };
 
+const withMetaSiteInfo = (HelmetComponent: CT) => (props: any) => {
+  const { children, ...rest } = props;
+  const siteInfo = btoa(JSON.stringify(useGitContext()));
```

Comment:
btoa creates base64 encoded string on client side. This obfuscate of data could be moved to backend.